src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1388,6 +1373,53 @@
       "required": ["message"]
     },
 
+    "resultProvenance": {
```

Comment:
>resultProvenance [](start = 5, length = 16)

where's the property bag for this type? don't forget to add the corresponding codegen hint or build will break.

Reply:
Good catch!

---
In reply to: [244648486](https://github.com/Microsoft/sarif-sdk/pull/1193#discussion_r244648486) [](ancestors = 244648486)